tests/libYARP_OS/NetworkTest.cpp

```diff
@@ -105,6 +105,10 @@ class NetworkTest : public UnitTest {
         checkFalse(Network::connect("/p1","/p3"),"bad connect, not existing destination");
         checkFalse(Network::connect("/p1","/p2 /p3"),"bad connect, invalid destination");
         checkFalse(Network::connect("/p1 /p2","/p2"),"bad connect, invalid source");
+        checkFalse(Network::connect("/p1/", "/p2"),"bad connect, source with ending '/'");
+        checkFalse(Network::connect("/p1", "/p2/"),"bad connect, destination with ending '/'");
+        checkFalse(Network::connect("p1", "/p2"),"bad connect, source without starting '/'");
+        checkFalse(Network::connect("/p1", "p2"),"bad connect, destination without starting '/'");
```

Comment:
It would be nice if you could add a new test, just for the `isValidPortName` method...